### DISCUSSION TITLE
Migration Help

### DISCUSSION THREAD

**Discussion by mcconnon12** [swellstores/community#8] [Help] [answered]
Hey All,

I'm looking for someone who can teach me how to migrate customers from one platform over to Swell. 

These will be subscriptions. Willing to pay for time. 

Please get in touch if you're interested.

Thanks,
Shawn

**Comment by logeshswell** [comment, marked as answer]
Hi @mcconnon12, Logesh here, Developer Evangelist at Swell.
I am glad to help you out with the customer data migration.
Please share some more context on the source platform or customer schema, so that I can guide you to map it with the Swell [customer](https://developers.swell.is/backend-api/accounts/the-account-model) schema.

**Reply by logeshswell** [reply]
@mcconnon12 cc: @ericingram You can use our migration guide to migrate your customer data to Swell.
https://developers.swell.is/guides/migrate-to-swell

**Reply by mcconnon12** [reply]
@logeshswell Hi, I would be transferring from Thrivecart which has an API and I can also download the data. All of the customers are already on Stripe. I would need some hand-holding with the migration.

**Reply by logeshswell** [reply]
Awesome, then you are good to use the same migration guide following the [customer section](https://developers.swell.is/guides/migrate-to-swell#2-customers).
@mcconnon12 Please feel free to drop a message in this thread if the docs are not clear enough or if you feel stuck on something on the migration, I am happy to guide you.

**Reply by mcconnon12** [reply]
@logeshswell Hi, I appreciate it. I'll probably have to hire someone who can do the migration. It would be great if Swell could develop a CSV migration tool. Is that on the radar?

**Reply by logeshswell** [reply]
@mcconnon12 Swell has a native solution to upload customers via CSV. Sharing you the screenshot for reference below:

<img width="1252" alt="Screenshot 2022-12-13 at 8 18 26 PM" src="https://user-images.githubusercontent.com/100766131/207365463-37758bc3-0292-43c8-a7c6-0e69d209308e.png">

**Reply by mcconnon12** [reply]
@logeshswell can it transfer the stripe token/payment info?

**Reply by mcconnon12** [reply]
@logeshswell or is there a process to do them one at a time so I can test the migration?

**Reply by logeshswell** [reply]
> Can it transfer the stripe token/payment info?

CSV import has some limitations on the fields that you want to import.

@mcconnon12 I would recommend you use our [batch API](https://developers.swell.is/guides/migrate-to-swell#swell-s-migration-endpoint) to migrate customer data considering your use case.

**Comment by ericingram** [comment]
There's a migration script somewhere, although it's not fully complete - @logeshswell can you find and link that here?
